docs/website/docs/user-guides/advanced/connecting-a-service.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Connecting to a Service
+sidebar_position: 1
+---
+
+Building on top of the [Go quickstart guide](../quickstart/go.md), this guide will connect a service to the Go application.
+
+## Prerequisites
+1. [Install the Service Binding Operator via Operator Hub](https://operatorhub.io/operator/service-binding-operator). [Read why this is required.](../../command-reference/add-binding.md#description)
+
+2. [Install Percona Server Mongodb Operator via Operator Hub](https://operatorhub.io/operator/percona-server-mongodb-operator).
+:::note
+The operator will be installed in a new namespace called "my-percona-server-mongodb-operator" and the service can only be deployed in this namespace.
```

Comment:
```suggestion
The operator will be installed in a new namespace called "my-percona-server-mongodb-operator". the service can only be deployed in this namespace.
```

Still we don't explain why this namespace :(, bit confusing IMO.

I think we should actually show to use `kubectl` commands? Because then we can specify what namespace to install this.

```

2. [Install Percona Server Mongodb Operator via Operator Hub](https://operatorhub.io/operator/percona-server-mongodb-operator).

Install to the same namespace as your component:

kubectl create -n mynamespace -f https://operatorhub.io/install/percona-server-mongodb-operator.yaml

```

Obviously not "mynamespace" but something else?

This explains to the user that the service will be in a specific namespace, etc at the beginning of the doc.